docs/conversion/s2u_features/S2U-46-mysql.sql

```diff
@@ -0,0 +1,38 @@
+-- S2U-46 --
+CREATE TABLE `mc_site_synchronization` (
+  `id` varchar(99) NOT NULL,
+  `site_id` varchar(255) NOT NULL,
+  `team_id` varchar(255) NOT NULL,
+  `forced` bit(1) DEFAULT NULL,
+  `status` int DEFAULT NULL,
```

Comment:
Umm, I kinda don't like this name since it's a reserved word

https://dev.mysql.com/doc/mysqld-version-reference/en/keywords-8-0.html